src/backend/vulkan/src/lib.rs

```diff
@@ -106,6 +85,12 @@ pub struct Instance {
 
     /// Supported extensions of this instance.
     pub extensions: Vec<&'static CStr>,
+
+    #[cfg(not(feature = "use-rtld-next"))]
+    pub entry: Entry,
```

Comment:
I wonder if this could be something like `dyn EntryTrait` that works for both? Doesn't matter much, just an option if we can remove this cfg branch